collection/lens.rkt

```diff
@@ -39,6 +67,21 @@
   (make-lens (λ (target) (nth target index))
              (λ (target value) (set-nth target index value))))
 
+(define (mapper-lens lens)
```

Comment:
Yes actually, now that I think about it. Except they sound a bit weird. I don't know. But to me, putting `er` on the end implies currying, which would be consistent across `mapper`, `taker`, and `dropper`.